client/src/components/AddNewList.jsx

```diff
@@ -4,9 +4,14 @@ import {
   Grid,
   makeStyles,
   Typography,
+  Dialog,
+  DialogTitle,
+  Button,
+  TextField,
 } from "@material-ui/core";
-import React from "react";
+import { Fragment, useState } from "react";
```

Comment:
If you use the [newer `Fragment` syntax](https://reactjs.org/docs/fragments.html#short-syntax), you don't even have to import it.